pkgs/by-name/cy/cyrus-imapd/package.nix

```diff
@@ -0,0 +1,191 @@
+{
+  autoreconfHook,
+  bison,
+  brotli,
+  coreutils,
+  cunit,
+  cyrus-imapd,
+  cyrus_sasl,
+  fetchFromGitHub,
+  fig2dev,
+  flex,
+  icu,
+  jansson,
+  lib,
+  libbsd,
+  libcap,
+  libchardet,
+  libical,
+  libmysqlclient,
+  libsrs2,
+  libuuid,
+  libxml2,
+  makeWrapper,
+  nghttp2,
+  openssl,
+  pcre2,
+  perl,
+  pkg-config,
+  postgresql,
+  rsync,
+  shapelib,
+  sqlite,
+  stdenv,
+  testers,
+  unixtools,
+  valgrind,
+  withCalalarmd ? true,
+  withHttp ? true,
+  withJMAP ? true,
+  withMySQL ? false,
+  withNNTP ? false,
+  withPgSQL ? false,
+  withSQLite ? true,
+  withXapian ? true,
+  wslay,
+  xapian,
+  zlib,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "cyrus-imapd";
+  version = "3.8.2";
+
+  src = fetchFromGitHub {
+    owner = "cyrusimap";
+    repo = "cyrus-imapd";
+    rev = "cyrus-imapd-${finalAttrs.version}";
+    hash = "sha256-5I4EaLnxu2M1a79XmxUh1PXq703EfI73/9aohKg7jyc=";
+  };
+
+  nativeBuildInputs = [
+    makeWrapper
+    pkg-config
+    autoreconfHook
+  ];
+  buildInputs =
+    [
```

Comment:
How about consistency in the syntax? Check `nativeBuildInputs` and `buildInputs` please.

Reply:
Oh great! It's good to know that you used `nixfmt-rfc-style`, you can ignore my comment.

IMHO, this kind of things should not happen, but OK...